scripts/superdesk-authoring/authoring.js

```diff
@@ -1646,7 +1646,9 @@
                 limit: '=',
                 html: '@'
             },
-            template: '<span class="char-count" ng-class="{error: numChars > limit}" translate>{{numChars}}/{{ limit }} characters </span>',
+            template: '<span class="char-count" ng-class="{error: limit && numChars > limit}" translate> characters</span>' +
+                    '<span class="char-count" ng-class="{error: limit && numChars > limit}">{{numChars}}' +
+                    '<span ng-if="limit" ng-class="{error: limit && numChars > limit}">/{{ limit }}</span></span>',
```

Comment:
at this point I would create a template file.
This is what I do when I have more than 1 line.